### PR TITLE
Add type mobileEmulationEnabled

### PR DESCRIPTION
**Pull request by tadashi0713**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: scripts/templates/webdriver.tpl.d.ts [hunk 1]
```diff
@@ -195,6 +195,7 @@ declare namespace WebDriver {
         // Chrome specific
         chromeOptions?: ChromeOptions;
         'goog:chromeOptions'?: ChromeOptions;
+        mobileEmulationEnabled: boolean;
```
Comment (mgrybyk) [inline scripts/templates/webdriver.tpl.d.ts]: Make it optional

Comment (tadashi0713) [reply scripts/templates/webdriver.tpl.d.ts]: Thanks 🙇

Comment (tadashi0713) [reply scripts/templates/webdriver.tpl.d.ts]: Modified @mgrybyk

**Comment by tadashi0713** [comment]
@christian-bromann 
Mobile emulation using Chromedriver is documented below.
http://chromedriver.chromium.org/mobile-emulation

I could only find Japanese document which wrote capability about `mobileEmulationEnabled` 🤔 
http://www.seleniumqref.com/api/java/about/About_capabilities.html

When mobile emulation is enabled, such capabilities will return

```
{ acceptInsecureCerts: false,
  acceptSslCerts: false,
  applicationCacheEnabled: false,
  browserConnectionEnabled: false,
  browserName: 'chrome',
  chrome:
   { chromedriverVersion: '2.43.600229 (3fae4d0cda5334b4f533bede5a4787f7b832d052)',
     userDataDir: '/var/folders/p7/0kyn0dx51z7bb1z5mhh6kqqh0000gq/T/.org.chromium.Chromium.wjiQdp' },
  cssSelectorsEnabled: true,
  databaseEnabled: false,
  'goog:chromeOptions': { debuggerAddress: 'localhost:63537' },
  handlesAlerts: true,
  hasTouchScreen: true,
  javascriptEnabled: true,
  locationContextEnabled: true,
  mobileEmulationEnabled: true,
  nativeEvents: true,
  networkConnectionEnabled: false,
  pageLoadStrategy: 'normal',
  platform: 'Mac OS X',
  rotatable: false,
  setWindowRect: true,
  takesHeapSnapshot: true,
  takesScreenshot: true,
  unexpectedAlertBehaviour: '',
  version: '72.0.3626.119',
  webStorageEnabled: true,
  'webdriver.remote.sessionid': '62d5bebbba1dd772284880666c11dcdf' }
[0-0] { acceptInsecureCerts: false,
  acceptSslCerts: false,
  applicationCacheEnabled: false,
  browserConnectionEnabled: false,
  browserName: 'chrome',
  chrome:
   { chromedriverVersion: '2.43.600229 (3fae4d0cda5334b4f533bede5a4787f7b832d052)',
     userDataDir: '/var/folders/p7/0kyn0dx51z7bb1z5mhh6kqqh0000gq/T/.org.chromium.Chromium.1nv8Xl' },
  cssSelectorsEnabled: true,
  databaseEnabled: false,
  'goog:chromeOptions': { debuggerAddress: 'localhost:63534' },
  handlesAlerts: true,
  hasTouchScreen: false,
  javascriptEnabled: true,
  locationContextEnabled: true,
  mobileEmulationEnabled: false,
  nativeEvents: true,
  networkConnectionEnabled: false,
  pageLoadStrategy: 'normal',
  platform: 'Mac OS X',
  rotatable: false,
  setWindowRect: true,
  takesHeapSnapshot: true,
  takesScreenshot: true,
  unexpectedAlertBehaviour: '',
  version: '72.0.3626.119',
  webStorageEnabled: true,
  'webdriver.remote.sessionid': '56443eb96ccf478e7c216c1cffa1d865' }
```

**Pull request merged**